portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DeleteSegments.testcase

```diff
@@ -611,28 +603,20 @@ definition {
 			ACIndividualsDashboard.assertIndividualAttribute(individualAttribute = "New Segment Test");
 		}
 
-		task ("Delete the CSV Data Source") {
-			ACSettings.deleteDataSource(
-				dataSourceName = "CSV Job Title",
-				dataSourceType = "CSV");
+		task ("Delete the Data Source") {
+			ACSettings.deleteDataSource(dataSourceName = "${dataSource}");
 		}
 
 		task ("Exit Settings") {
 			ACNavigation.exitSettings();
 		}
 
-		task ("Assert the segment doesn't display under the contact user") {
-			ACNavigation.goToIndividuals();
-
-			ACNavigation.switchTab(tabName = "Known Individuals");
-
-			ACIndividualsDashboard.goToKnownIndividualDetails(individualName = "ac ac");
+		task ("Open the created segment and assert the user is not present") {
+			ACNavigation.goToSegments();
 
-			ACNavigation.switchTab(tabName = "Segments");
+			ACSegments.accessSegment(segmentName = "New Segment Test");
 
-			AssertElementNotPresent(
-				key_individualAttribute = "New Segment Test",
-				locator1 = "ACIndividualsDashboard#KNOWN_INDIVIDUAL_ATTRIBUTE");
+			ACUtils.viewNameListIsNotPresent(nameList = "ac ac");
```

Comment:
Hey @manuferreira, this macro will not work if the result of the list is an empty state message. This path waits for a format with classes that only appear when there is a list on the page, if you had another user appearing in the list this macro would work, but as it will display the empty state message, so I think you could use:

`ACUtils.viewGenericText(textValueList = "There are no members found on the selected time period.");`